### PR TITLE
Fix Op (aten::native_batch_norm) | feat(torchlib)

### LINKED ISSUE #1791: [Pattern] BatchNorm training=True -> False
torch.export sometimes creates BatchNorm with training=True even though the model is an inference model. Create a rewrite rule to set training to False.

Related: https://github.com/microsoft/onnxscript/issues/1338

### LINKED ISSUE #1803: BatchNorm training gets scalar `running_mean` and `running_var`
Discovered in bench with the command:
```
python -m bash_bench.bash_bench_nvidia --device cuda --dtype float16 --quiet 0 --memory_peak 1 --exporter torch-onnx --model Generator -r 30 -w 10 --verbose 1 --dump_ort 1
```

Error:
```
  File "/root/onnxruntime/build/Linux/RelWithDebInfo/onnxruntime/capi/onnxruntime_inference_collection.py", line 480, in _create_inference_session
    sess = C.InferenceSession(session_options, self._model_path, True, self._read_config_from_model)
onnxruntime.capi.onnxruntime_pybind11_state.Fail: [ONNXRuntimeError] : 1 : FAIL : Load model from dump_bash_bench/Generator-torch-onnx-cuda-torch.float16/model.onnx failed:Node (node__aten_native_batch_norm_training_onnx_95) Op (_aten_native_batch_norm_training_onnx) [ShapeInferenceError] Inference error(s): (op_type:BatchNormalization, node name: n0): [ShapeInferenceError] Input 3 expected to have rank 1 but has rank 0 in node BatchNormalization (n0).
```

### PR DESCRIPTION
**Pull request by titaiwangms**
Fix #1803 
Fix #1791 
Add the conversion of scalar to tensor

### CODE REVIEW

**Review by justinchuby — APPROVED** [review]

**Comment by titaiwangms** [comment]
Looks like we need to hack the training tests as well.
